tcms/templates/run/status_statistics.html

```diff
@@ -6,7 +6,8 @@
 	<ul class="count-per-status-container">
 		<li>
 			<label>{% trans "TOTAL" %}</label>
-			[<a href="#" class="js-caserun-total"></a>]
+			{% comment %} empty query param means no filter {% endcomment %}
+			[<a href="?status_id" class="js-caserun-total"></a>]
```

Comment:
You are not passing any value to this parameter so it is useless. Filtering TOTAL is the same as reloading the TR page.  Also remove the class attribute if it is no-longer needed.

Reply:
we need a class, because when we draw the chart we get the element by class and write the number of executions in that element. I can change the name, because it does not make much sense now